Source/WebKit/WebProcess/WebPage/RemoteLayerTree/PlatformCALayerRemote.mm

```diff
@@ -290,6 +290,35 @@
     return owner() && owner()->platformCALayerContainsBitmapOnly(this);
 }
 
+DestinationColorSpace PlatformCALayerRemote::displayColorSpace() const
+{
+    switch (contentsFormat()) {
+    case ContentsFormat::RGBA8:
+        break;
+
+#if HAVE(IOSURFACE_RGB10)
+    case ContentsFormat::RGBA10:
+#if PLATFORM(IOS_FAMILY)
```

Comment:
should we use `HAVE(CORE_GRAPHICS_EXTENDED_SRGB_COLOR_SPACE)` instead or is there a reason it is iOS specific?

Reply:
We have this definition for `HAVE_IOSURFACE_RGB10`
```
#if PLATFORM(IOS_FAMILY) && !PLATFORM(MACCATALYST) && !PLATFORM(IOS_FAMILY_SIMULATOR)
#define HAVE_IOSURFACE_RGB10 1
#endif
```
And this definition for `HAVE_CORE_GRAPHICS_EXTENDED_SRGB_COLOR_SPACE`
```
#if PLATFORM(COCOA)
#define HAVE_CORE_GRAPHICS_EXTENDED_SRGB_COLOR_SPACE 1
#endif
```

So there is no need for either `PLATFORM(IOS_FAMILY)` or `HAVE(CORE_GRAPHICS_EXTENDED_SRGB_COLOR_SPACE)`